Cargo.toml

```diff
@@ -39,3 +40,15 @@ tracing = "0.1"
 tracing-subscriber = "0.3"
 tree_hash = "0.9.0"
 tree_hash_derive = "0.9.0"
+discv5 = "0.9.0"
+tokio = { version = "1.42.0", features = ["rt", "rt-multi-thread", "sync", "signal", "time","macros"] }
+log = "0.4.2"
```

Comment:
```suggestion
```

we are using tracing instead of log

can you sort these alphabetically excluding the # local dependencies section